app/models/TradeAgreement/TradeAgreement.scala

```diff
@@ -0,0 +1,8 @@
+package models.TradeAgreement
+
+case class TradeAgreement(
+    buyer: Dealer,
+    seller: Dealer,
+    deliveryTypeCode: String,
```

Comment:
อันนี้จะให้หน้าบ้านกรอกอะไรอ่ะ

Reply:
อ้างอิงจาก requirement นะครับ เข้าใจว่าเป็น code 3 ตัวอักษร

แนวทางการระบุเงื่อนไขการซื้อขาย (Trade Term Code) 
เงื่อนไขการซื้อขาย (3.1.3.1) สำมารถอ้างอิงได้จากรายการรหัส UNECE Recommendation No. 5 
Abbreviations of INCOTERMS http://tfig.unece.org/contents/recommendation-5.htm

Reply:
จะให้หน้าบ้านกรอกเป็น code เลยหรอ

Reply:
คิดว่าอาจจะให้หน้าบ้านทำเป็น dropdown ครับ

Reply:
อ่าเช